metrics/web/docs/features/remote_config/diagrams/remote_config_presentation_class.puml

```diff
@@ -0,0 +1,27 @@
+@startuml
+
+package base.domain.usecases {
+    class GetRemoteConfigUseCase{}
+}
+
+package common.presentation.state {
+    class RemoteConfigNotifier {
+        + isLoginFormEnabled : bool
+        + isFpsMonitorEnabled : bool
+        + isRendererDisplayEnabled : bool
```

Comment:
Not sure we should expose all fields of the config class. I suspect we should use a class instance itself since any changes to the config model will require changes to the notifier.